README.md

```diff
@@ -36,14 +36,17 @@ Then configure the rules you want to use under the rules section.
 ```json
 {
     "rules": {
-        "hss-i18n/no-interpolate-literal": "error"
+        "hss-i18n/no-interpolate-literal": "error",
+        "hss-i18n/gettext-literal": "error"
```

Comment:
An alternate name to consider might be `require-gettext-literal`, based upon some other rules I've come across, but the name is still fine as-is.